### DISCUSSION TITLE
Module Federation without Lerna. How to work with two different repositories

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [webpack/webpack#13156] [Q&A]
Hello. Interested in the question of how to set up paths to shared files between projects that are in different repositories. In this case, it is generally possible to build a microfronted, or everything needs to be kept in monorepos from Lerna

**Comment by alexander-akait** [comment]
You can use multi compiler mode https://github.com/webpack/webpack/tree/master/examples/multi-compiler and use different dirs

**Reply by popuguytheparrot** [reply]
Мulti compiler mode for what? The source code for the second application is in a different repository. I don't understand a bit how I can link the build of the second application

**Reply by alexander-akait** [reply]
You can use https://git-scm.com/book/en/v2/Git-Tools-Submodules
